config/conf.json

```diff
@@ -31,7 +31,7 @@
             "base_label": "Hail",
             "tools": ["python", "spark"],
             "packages": { "python": ["hail"] },
-            "version": "0.0.15",
+            "version": "0.0.16",
```

Comment:
Note hail was bumped to `0.0.16` in https://github.com/DataBiosphere/terra-docker/pull/138 -- you may want to let that one merge first them bump it to `0.0.17` here